src/FormControl/FormControl.tsx

```diff
@@ -23,14 +24,14 @@ export type FormControlProps = {
    * If true, the user must specify a value for the input before the owning form can be submitted
    */
   required?: boolean
-}
+} & SxProp
 
 export interface FormControlContext extends Pick<FormControlProps, 'disabled' | 'id' | 'required'> {
   captionId: string
   validationMessageId: string
 }
 
-const FormControl = ({children, disabled, id: idProp, required}: FormControlProps) => {
+const FormControl = ({children, disabled, id: idProp, required, sx}: FormControlProps) => {
```

Comment:
`FormControl.Label`, `FormControl.Validation`, and `FormControl.Caption`, `FormControl.LeadingVisual` should also accept the `sx` prop